internal/core/components/watcher.go

```diff
@@ -44,6 +47,11 @@ func (w *Watcher) Init() {
 		}
 		wg.Wait()
 	}
+
+	if !loaded {
```

Comment:
It seems that:
before this change, any load failure will break the process and return the error;
after this change, only all stores load failed will break the process and return the error

Reply:
Yes. But usually, when APISIX starts, /prefix/routes, /prefix/services and /prefix/upstreams will be pre-created, so it seems more reasonable to allow apisix-seed to start after all loading is successful, right?